momentum_scanner_intraday.py

```diff
@@ -57,6 +63,31 @@ def get_risk_query(token, stoploss=0.05, profittaking=0.05):
     print("enter at:", price, "stop-loss:", slprice, "profit-taking:", ptprice)
     return slprice, ptprice
 
+def lookup(dex):
+    """
+    find the chain on which a dex is deployed
+    """
+    chain_cg = None
+    chain_cgterminal = None
+    for i in DEX_CHAIN:
```

Comment:
Since `DEX_CHAIN` is now a dict, this can be a lookup instead of a for loop.

Reply:
You mean to rename it?

Reply:
No I mean `DEX_CHAIN` is a dict, do not iterate all its keys to find something.